weave-js/src/components/Panel2/panellib/stackinfo.ts

```diff
@@ -0,0 +1,26 @@
+import {Type} from '../../../core/model';
+import {IconName} from '../../Icon';
+import {getPanelCategory, getPanelIcon} from '../PanelRegistry';
+import {getStackIdAndName, PanelSpecNode} from './libpanel';
+import {PanelCategory} from './types';
+
+export type StackInfo = {
```

Comment:
Out of curiosity, do you know why it's called Stack? `PanelInfo` feels like it would be a better name.

Reply:
I was trying to be consistent with the current code terminology without having a great understanding of this myself. I think basically we generate options for how to render a type with panels. Each of those options is frequently just a single panel. But technically, each option suggested is actually a stack of panels that just frequently only has a single panel in it. There are cases like the "maybe" handling where the suggestion might be multiple panels that would be processed in order.